test/test_ci.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+
+from githubmarkdownui import ci
+from githubmarkdownui.inline import bold
+
+
+sample_job = ci.CIJob(ci.CIJobMetadata(ci.CIStatus.SUCCEEDED, 'my job'), [
+    ci.CITask(ci.CITaskMetadata(ci.CIStatus.SUCCEEDED, 'first task', '15s', 'additional info')),
+    ci.CITask(ci.CITaskMetadata(ci.CIStatus.SUCCEEDED, 'second task', '1m 29s', 'additional info')),
+], [
+    ci.CIJob(ci.CIJobMetadata(ci.CIStatus.SUCCEEDED, 'first child job'), [
+        ci.CITask(ci.CITaskMetadata(ci.CIStatus.SUCCEEDED, 'my first task', '10s', 'additional info')),
+    ]),
+    ci.CIJob(ci.CIJobMetadata(ci.CIStatus.FAILED, 'second child job'), [
+        ci.CITask(ci.CITaskMetadata(ci.CIStatus.SUCCEEDED, 'another task', '10s', 'additional info')),
+        ci.CITask(ci.CITaskMetadata(ci.CIStatus.FAILED, 'task 2 foobarbaz', info='additional info')),
+        ci.CITask(ci.CITaskMetadata(ci.CIStatus.SUCCEEDED, 'third task', '1m 8s')),
+        ci.CITask(ci.CITaskMetadata(ci.CIStatus.FAILED, 'another failed task', '1h 20m', 'additional info')),
+    ]),
+])
+
+
+def test_get_left_justified_task_string_length():
+    assert sample_job.tasks[0].get_left_justified_task_string_length() == len('✅  first task (15s)')
+    assert sample_job.tasks[1].get_left_justified_task_string_length() == len('✅  second task (1m 29s)')
```

Comment:
Why don't you use the emoji from the defined enum `CIStatus`? If they're changed for whatever reason, you'll have a fun time changing these tests right?  😅

Reply:
good idea